File: src/main/java/br/com/jiratorio/domain/request/UpdateUserConfigRequest.java
package br.com.jiratorio.domain.request;

import br.com.jiratorio.domain.ChartType;
import lombok.Data;

@Data
public class UpdateUserConfigRequest {

    private String state;

    private String city;

    private String holidayToken;

    private ChartType leadTimeChartType;

    private ChartType throughputChartType;

}

File: src/main/kotlin/br/com/jiratorio/domain/request/UpdateUserConfigRequest.kt
package br.com.jiratorio.domain.request

import br.com.jiratorio.domain.chart.ChartType

data class UpdateUserConfigRequest(

    val state: String? = null,

    val city: String? = null,

    val holidayToken: String? = null,

    val leadTimeChartType: ChartType? = null,

    val throughputChartType: ChartType? = null

)
